Dockerfile

```diff
@@ -1 +1,12 @@
-FROM gliderlabs/alpine:3.3
+FROM openjdk:7
```

Comment:
could use `openjdk:7-jre-alpine` or `openjdk:7-jre` (for debian). JDK is only required when building java apps, JRE includes only the runtime.